log/levels/levels.go

```diff
@@ -0,0 +1,127 @@
+package levels
+
+import "github.com/go-kit/kit/log"
+
+// Levels provides a leveled logging wrapper around a logger. It has five
+// levels: debug, info, warning (warn), error, and critical (crit). If you
```

Comment:
I was expecting we would add support for something along those lines in the future. Before we cram it all into `levels.Levels` though, consider if it can be achieved with a level aware `log.Logger`.